### PR TITLE
2 singup and dbschma created

### PR DESCRIPTION
**Pull request by Ridham021**
Register user Rest  and DbSchema created

### CODE REVIEW

File: apmcSpringBoot/src/main/java/com/apmc/apmcSpringBoot/security/config/CustomUserDetailService.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        if(user==null){
+            throw new UsernameNotFoundException("User not found");
+        }
+        else{
+            return MyUserDetails.build(user.get());
+        }}catch (Exception e){
+            e.printStackTrace();
+//            return null;
+        }finally {
+            System.out.println("Finnaly is called");
```
Comment (nihar30-dev) [inline apmcSpringBoot/src/main/java/com/apmc/apmcSpringBoot/security/config/CustomUserDetailService.java:33]: Remove  System.out.println

File: apmcSpringBoot/src/main/java/com/apmc/apmcSpringBoot/security/config/WebConfig.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
+//package com.apmc.apmcSpringBoot.security.config;
```
Comment (nihar30-dev) [inline apmcSpringBoot/src/main/java/com/apmc/apmcSpringBoot/security/config/WebConfig.java:1]: remove commented file or code

File: apmcSpringBoot/src/main/java/com/apmc/apmcSpringBoot/security/jwt/JwtUtils.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+import java.util.Date;
+
+@Component
+public class JwtUtils {
+
+    private static final Logger logger = LoggerFactory.getLogger(JwtUtils.class);
+
+//    @Value("${app.jwtSecret}")
+    private String jwtSecret="secretKeyssssssssssssssssssssssssssssssssfdfffffffffffdfdfdfdfdfbnnnnnnnnnnnnnnnnnnnmbhjjjjjjjjjjjjjhhhhhhhhhhhhhhhhhgfgjhghgjgjggjgjgjhgytfytjyfkfj";
```
Comment (nihar30-dev) [inline apmcSpringBoot/src/main/java/com/apmc/apmcSpringBoot/security/jwt/JwtUtils.java:19]: add this in properties file

**Review by nihar30-dev — COMMENTED** [review]
Try to add method description when you get some time it will help in long run as you need to fresh your concepts.

File: apmcSpringBoot/src/main/java/com/apmc/apmcSpringBoot/security/config/CustomUserDetailService.java (hunk 1 above)
Comment (Ridham021) [reply apmcSpringBoot/src/main/java/com/apmc/apmcSpringBoot/security/config/CustomUserDetailService.java:33]: resolved

File: apmcSpringBoot/src/main/java/com/apmc/apmcSpringBoot/security/config/WebConfig.java (hunk 2 above)
Comment (Ridham021) [reply apmcSpringBoot/src/main/java/com/apmc/apmcSpringBoot/security/config/WebConfig.java:1]: resolved

File: apmcSpringBoot/src/main/java/com/apmc/apmcSpringBoot/security/jwt/JwtUtils.java (hunk 3 above)
Comment (Ridham021) [reply apmcSpringBoot/src/main/java/com/apmc/apmcSpringBoot/security/jwt/JwtUtils.java:19]: resolved

**Review by nihar30-dev — APPROVED** [review]

**Pull request merged**